pkg/dhclient/dhclient.go

```diff
@@ -40,6 +59,15 @@ func IfUp(ifname string) (netlink.Link, error) {
 		}
 
 		if iface.Attrs().Flags&net.FlagUp == net.FlagUp {
+			// For ipv6, we cannot bind to the port until Duplicate
+			// Address Detection (DAD) is complete which is
+			// indicated by the link being no longer marked as
+			// "tentative".
+			if ipv6 && !isIpv6LinkReady(ifname, iface) {
```

Comment:
This doesn't work so well when the interface has available ipv4 addresses, but not ipv6 addresses.